android/app/src/main/java/com/mattermost/rnbeta/ReceiptDelivery.java

```diff
@@ -26,6 +26,9 @@
 public class ReceiptDelivery {
     static final String CURRENT_SERVER_URL = "@currentServerUrl";
 
+    private static int reRequestCount = 0;
```

Comment:
This is never being reset to 0, maybe just pass it as an argument of `makeServerRequest`

Reply:
Done via e41ad52d855c958b7405801d2b62ededf785eddb